### PR TITLE
Bank account category

### PR DESCRIPTION
**Pull request by igorsergiichuk**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem

Bank account category Model Test was missing

## How I resolved it

Added Bank account category Model Test + mocks

## How to verify it

Test it!

Notify the following people: @darrynten

### CODE REVIEW

File: tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json [hunk 1]
```diff
@@ -1 +1,6 @@
-
+{
+  "Description": "sample string 1",
+  "ID": 2,
+  "Modified": "2017-07-25",
+  "Created": "2017-07-25"
+}
```
Comment (darrynten) [inline tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json]: Which is correct? The red or the green?

**Review by darrynten — COMMENTED** [review]
Not sure what's going on here, which one is correct?

File: tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json (hunk 1 above)
Comment (igorsergiichuk) [reply tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json]: green is correct. SAGE docs give us a blank response. @fergusdixon contacted them. They wrote a letter which indicates that RESP is the same as REQ.

**Review by darrynten — DISMISSED** [review]

**Comment by darrynten** [comment]
These seem to be failing on Travis due to missing mocks

screen shot 2017-08-04 at 6 37 34 pm

File: tests/SageOne/Models/BankAccountCategoryModelTest.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            $this->assertEquals($model->id, 2);
+            $this->assertEquals($model->modified->format('Y-m-d'), '2017-07-25');
+            $this->assertEquals($model->created->format('Y-m-d'), '2017-07-25');
+        });
+    }
+
+    public function testSave()
+    {
+        $this->verifySave(BankAccountCategory::class, function ($model) {
+            $model->description = "sample string 1";
```
Comment (darrynten) [inline tests/SageOne/Models/BankAccountCategoryModelTest.php]: This should be single quotes :) Same in the next method

**Review by darrynten — CHANGES_REQUESTED** [review]
A double quote that should be single

**Review by darrynten — APPROVED** [review]
Nice!

**Pull request merged**